docs/source/code.rst

```diff
@@ -0,0 +1,60 @@
+Code Guidelines
```

Comment:
@elsevers 
We should discuss this in the meeting on 10/11, and if this spills over I think this is something you and I should go through together. It may be relatively time consuming, but it seems that this is the crux of creating all of this documentation in the first place.

Reply:
I am resolving this. This issue has been addressed and changes have been made. More feedback will be welcome regarding this file once a re-review is requested.